### PR TITLE
Client device modifications 

### PR DESCRIPTION
**Pull request by legoeruro**
Sorry, I forgot to commit and push before the last PR.

### CODE REVIEW

File: unity/Assets/Scripts/ARFlow/ARFlowClient.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 using System;
 using Cysharp.Net.Http;
 using Grpc.Net.Client;
+using Unity.VisualScripting.FullSerializer;
```
Comment (YiqinZhao) [inline unity/Assets/Scripts/ARFlow/ARFlowClient.cs:4]: Why is this package introduced? For the `print` function?

File: unity/Assets/Scripts/ARFlowXiheDemo.cs [hunk 2]
```diff
@@ -77,6 +77,7 @@ private void UpdatePlacementIndicator()
 
     private void UpdatePlacementPose()
     {
+        print(Camera.current);
```
Comment (YiqinZhao) [inline unity/Assets/Scripts/ARFlowXiheDemo.cs:80]: Why do we need this print?

**Review by YiqinZhao — COMMENTED** [review]
Thanks, see the comments

File: unity/Assets/Scripts/ARFlow/ARFlowClient.cs (hunk 1 above)
Comment (legoeruro) [reply unity/Assets/Scripts/ARFlow/ARFlowClient.cs:4]: It was included when I mistyped something, I'll remove it

File: unity/ProjectSettings/EditorBuildSettings.asset [hunk 3]
```diff
@@ -5,9 +5,12 @@ EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
   m_Scenes:
-  - enabled: 1
+  - enabled: 0
     path: Assets/Scenes/DeviceData.unity
     guid: a85ce1c6384d24c2a8e016dbf92ebe5c
+  - enabled: 1
+    path: Assets/Scenes/XiheDemo.unity
+    guid: 16bf2390a4afb41c48a28fbea06c2cdd
```
Comment (YiqinZhao) [inline unity/ProjectSettings/EditorBuildSettings.asset:13]: I think we should keep the default scene as `DeviceData.unity` on the main branch. Let's revert these changes back.

Comment (legoeruro) [reply unity/ProjectSettings/EditorBuildSettings.asset:13]: Sounds good, I'll make sure of this in later changes

**Review by YiqinZhao — APPROVED** [review]
LGTM
